demo/cue-events.html

```diff
@@ -0,0 +1,332 @@
+<!DOCTYPE html>
```

Comment:
We'll need to update this page, as I fixed a bug with the point/segment input fields in https://github.com/bbc/peaks.js/commit/5448e420bfb4e053a58f67136673af5afe67eb7b.